server.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+
+// Entry point or our application
+async function main() {
+	const app = express();
+
+	return app.listen(8080);
```

Comment:
replace hard tabs with spaces

Also, you need to add the call to `db.js` here and return the `client` for the handler below to work.

Finally, adding a log message to indicate the server has started would be a good addition

```suggestion
  const { db, client }  = await connect();
  
  const server = app.listen(8082, () => {
    console.log("Server is listening on port 8080");
  });

  return { server, client };
```

And don't forget to add `const connect = require("./db");` above.